Gemfile.lock

```diff
@@ -375,4 +381,4 @@ DEPENDENCIES
   xml-simple
 
 BUNDLED WITH
-   1.16.0
+   1.16.1
```

Comment:
Please use bundler 1.16.0

Reply:
Do I need to specify it in `Gemfile`? Or just downgrade my Bundler locally and `Gemfile.lock` will be changed? Also, is there any difference between the two versions?

Reply:
> just downgrade my Bundler locally and Gemfile.lock will be changed?
yes.
> is there any difference between the two versions?
changing bundler version here would mean we need to update bundler on our servers. we prefer to do that separately (not as part of any PR)

Reply:
I've got it done. Thanks